src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -98,9 +92,10 @@ the MIT License. See LICENSE in the project root for license information. -->
                 </ItemsControl.ItemTemplate>
             </ItemsControl>
 
+            <!-- Additional Colors -->
             <ItemsControl Grid.Row="2"
                           Grid.Column="1">
-                <StackPanel Orientation="Vertical" Height="59">
+                <StackPanel Height="59">

```

Comment:
No joke, 60 is too much. This aligns the 4 special colors (foreground, background, etc...) with the 4 rows on the left.

Reply:
1. This is a coincidence. Do not rely on coincidence.
2. I don't care why 59 was chosen, I care that there's at least four copies of it in this file :wink:

Reply:
Urgh. So I tried adding a style for StackPanel and TextBlock in the `Page.Resources` block, and I ended up with this:
![styled controls fail](https://user-images.githubusercontent.com/11050425/99299538-32a95000-2800-11eb-8fb9-6b0826b2a472.png)

```xml
        <Style TargetType="StackPanel">
            <Setter Property="Margin" Value="0,0,0,20"/>
            <Setter Property="Height" Value="59"/>
        </Style>

        <Style TargetType="TextBlock">
            <Setter Property="Margin" Value="10,0,0,10"/>
        </Style>
```

I don't see us merging resource dictionaries. Idk why it ruins all of it.

Also, the StackPanel styling is what makes the right column so ugly. The TextBlock styling seems to only apply to the one in the right column too? Help

Reply:
Perhaps because the ones on the left come from a DataTemplate...

Please stop applying styles to _entire types_ like StackPanel. That's never going to do what you want. :P It's going to apply to _every StackPanel_ in the page. Are you certain that no stackpanels were used for anything that wasn't a single-textblock-single-colored-button group?

Reply:
Even putting these in an `<ItemsControl.Resources>` for the right side does the same thing :/

> Please stop applying styles to _entire types_ like StackPanel. That's never going to do what you want. 

So... how do I apply styling to everything without (1) directly applying it to everything or (2) setting it as a style for the page? Is there a cleaner way to do this?

Reply:
> Even putting these in an `<ItemsControl.Resources>` for the right side does the same thing :/
> 
> > Please stop applying styles to _entire types_ like StackPanel. That's never going to do what you want.
> 
> So... how do I apply styling to everything without (1) directly applying it to everything or (2) setting it as a style for the page? Is there a cleaner way to do this?

Can you not do ``<StackPanel Style="{StaticResource NameOfCustomStyle}">``